### ISSUE TITLE
[Bug]: netdata crash when trying to stream from Jetson Nano to AMD64-pc

### ISSUE THREAD

**Issue by MaxMatti** [netdata/netdata#14016]
### Bug description

After setting up streaming according to https://learn.netdata.cloud/docs/agent/streaming#configuration the parent keeps crashing as soon as the child connects to it.

### Expected behavior

No crash

## Child config:

* `netdata.conf`
```
[global]
    run as user = netdata

    # default storage size - increase for longer data retention
    page cache size = 32
    dbengine multihost disk space = 256
    memory mode = none

[web]
    mode = none
    accept a streaming request every seconds = 0
```

* `stream.conf`
```
# -----------------------------------------------------------------------------
# 1. ON CHILD NETDATA - THE ONE THAT WILL BE SENDING METRICS

[stream]
    enabled = yes
    destination = 192.168.100.159
    ssl skip certificate verification = yes
    api key = 36739acc-fc72-4183-b2a5-9ea85404143e
    enable compression = yes
    timeout seconds = 60
    default port = 19999
    send charts matching = *
    buffer size bytes = 10485760
    reconnect delay seconds = 5
    initial clock resync iterations = 60

# -----------------------------------------------------------------------------
# 2. ON PARENT NETDATA - THE ONE THAT WILL BE RECEIVING METRICS

[API_KEY]
    enabled = no
    allow from = *
    default postpone alarms on connect seconds = 60

# -----------------------------------------------------------------------------
# 3. PER SENDING HOST SETTINGS, ON PARENT NETDATA
#    THIS IS OPTIONAL - YOU DON'T HAVE TO CONFIGURE IT

[MACHINE_GUID]
    enabled = no
    allow from = *
    postpone alarms on connect seconds = 60
```

## Parent config:
* `netdata.conf`
```
[global]
    run as user = netdata

    # default storage size - increase for longer data retention
    page cache size = 32
    dbengine multihost disk space = 256

[db]
    mode = dbengine

[plugins]
    perf = yes

[plugin:perf]
    update every = 1
    command options = all
```

* `stream.conf`
```
# -----------------------------------------------------------------------------
# 1. ON CHILD NETDATA - THE ONE THAT WILL BE SENDING METRICS

[stream]
    enabled = no
    destination =
    ssl skip certificate verification = yes
    api key =
    enable compression = yes 
    timeout seconds = 60
    default port = 19999
    send charts matching = *
    buffer size bytes = 10485760
    reconnect delay seconds = 5
    initial clock resync iterations = 60

# -----------------------------------------------------------------------------
# 2. ON PARENT NETDATA - THE ONE THAT WILL BE RECEIVING METRICS

[36739acc-fc72-4183-b2a5-9ea85404143e]
    enabled = yes
    allow from = *
    default memory mode = dbengine
    health enabled by default = auto
    default postpone alarms on connect seconds = 60
    enable compression = yes 
    enable replication = yes 
    seconds to replicate = 86400 
    replication_step = 600 

# -----------------------------------------------------------------------------
# 3. PER SENDING HOST SETTINGS, ON PARENT NETDATA
#    THIS IS OPTIONAL - YOU DON'T HAVE TO CONFIGURE IT

[MACHINE_GUID]
    enabled = no
    allow from = *
    memory mode = dbengine 
    postpone alarms on connect seconds = 60
    enable compression = yes 
    enable replication = yes 
    seconds to replicate = 86400 
    replication_step = 600 
```

### Installation method

kickstart.sh

#### Child:
```
$ uname -a; grep -HvE "^#|URL" /etc/*release
Linux devkit-ext-mstaff 4.9.253-tegra #1 SMP PREEMPT Wed Apr 20 14:25:12 PDT 2022 aarch64 aarch64 aarch64 GNU/Linux
/etc/lsb-release:DISTRIB_ID=Ubuntu
/etc/lsb-release:DISTRIB_RELEASE=18.04
/etc/lsb-release:DISTRIB_CODENAME=bionic
/etc/lsb-release:DISTRIB_DESCRIPTION="Ubuntu 18.04.6 LTS"
/etc/os-release:NAME="Ubuntu"
/etc/os-release:VERSION="18.04.6 LTS (Bionic Beaver)"
/etc/os-release:ID=ubuntu
/etc/os-release:ID_LIKE=debian
/etc/os-release:PRETTY_NAME="Ubuntu 18.04.6 LTS"
/etc/os-release:VERSION_ID="18.04"
/etc/os-release:VERSION_CODENAME=bionic
/etc/os-release:UBUNTU_CODENAME=bionic
```

Note: this is not the default Ubuntu, it's the version provided by Nvidia for the Jetson Nano. It can be downloaded [here](https://developer.nvidia.com/embedded/jetpack-sdk-462).

#### Parent:
```
$ uname -a; grep -HvE "^#|URL" /etc/*release
Linux anvajo-nb97.anvajo.local 5.14.0-1054-oem #61-Ubuntu SMP Fri Oct 14 13:05:50 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
/etc/lsb-release:DISTRIB_ID=Ubuntu
/etc/lsb-release:DISTRIB_RELEASE=20.04
/etc/lsb-release:DISTRIB_CODENAME=focal
/etc/lsb-release:DISTRIB_DESCRIPTION="Ubuntu 20.04.5 LTS"
/etc/os-release:NAME="Ubuntu"
/etc/os-release:VERSION="20.04.5 LTS (Focal Fossa)"
/etc/os-release:ID=ubuntu
/etc/os-release:ID_LIKE=debian
/etc/os-release:PRETTY_NAME="Ubuntu 20.04.5 LTS"
/etc/os-release:VERSION_ID="20.04"
/etc/os-release:VERSION_CODENAME=focal
/etc/os-release:UBUNTU_CODENAME=focal
```

#### Child:
```
$ netdata -W buildinfo
Version: netdata v1.36.0-364-nightly
Configure options:  '--build=aarch64-linux-gnu' '--includedir=${prefix}/include' '--mandir=${prefix}/share/man' '--infodir=${prefix}/share/info' '--disable-silent-rules' '--libdir=${prefix}/lib/aarch64-linux-gnu' '--libexecdir=${prefix}/lib/aarch64-linux-gnu' '--disable-maintainer-mode' '--prefix=/usr' '--sysconfdir=/etc' '--localstatedir=/var' '--libdir=/usr/lib' '--libexecdir=/usr/libexec' '--with-user=netdata' '--with-math' '--with-zlib' '--with-webdir=/var/lib/netdata/www' '--disable-dependency-tracking' '--disable-ebpf' 'build_alias=aarch64-linux-gnu' 'CFLAGS=-g -O2 -fdebug-prefix-map=/usr/src/netdata=. -fstack-protector-strong -Wformat -Werror=format-security' 'LDFLAGS=-Wl,-Bsymbolic-functions -Wl,-z,relro' 'CPPFLAGS=-Wdate-time -D_FORTIFY_SOURCE=2' 'CXXFLAGS=-g -O2 -fdebug-prefix-map=/usr/src/netdata=. -fstack-protector-strong -Wformat -Werror=format-security'
Install type: binpkg-deb
    Binary architecture: aarch64
    Packaging distro:  
Features:
    dbengine:                   YES
    Native HTTPS:               YES
    Netdata Cloud:              YES 
    ACLK:                       YES
    TLS Host Verification:      YES
    Machine Learning:           YES
    Stream Compression:         NO
Libraries:
    protobuf:                YES (system)
    jemalloc:                NO
    JSON-C:                  YES
    libcap:                  NO
    libcrypto:               YES
    libm:                    YES
    tcalloc:                 NO
    zlib:                    YES
Plugins:
    apps:                    YES
    cgroup Network Tracking: YES
    CUPS:                    YES
    EBPF:                    NO
    IPMI:                    YES
    NFACCT:                  YES
    perf:                    YES
    slabinfo:                YES
    Xen:                     NO
    Xen VBD Error Tracking:  NO
Exporters:
    AWS Kinesis:             NO
    GCP PubSub:              NO
    MongoDB:                 NO
    Prometheus Remote Write: YES
Debug/Developer Features:
    Trace Allocations:       NO
```

#### Parent:
```
$ netdata -W buildinfo
Version: netdata v1.36.0-364-nightly
Configure options:  '--build=x86_64-linux-gnu' '--includedir=${prefix}/include' '--mandir=${prefix}/share/man' '--infodir=${prefix}/share/info' '--disable-silent-rules' '--libdir=${prefix}/lib/x86_64-linux-gnu' '--libexecdir=${prefix}/lib/x86_64-linux-gnu' '--disable-maintainer-mode' '--prefix=/usr' '--sysconfdir=/etc' '--localstatedir=/var' '--libdir=/usr/lib' '--libexecdir=/usr/libexec' '--with-user=netdata' '--with-math' '--with-zlib' '--with-webdir=/var/lib/netdata/www' '--disable-dependency-tracking' 'build_alias=x86_64-linux-gnu' 'CFLAGS=-g -O2 -fdebug-prefix-map=/usr/src/netdata=. -fstack-protector-strong -Wformat -Werror=format-security' 'LDFLAGS=-Wl,-Bsymbolic-functions -Wl,-z,relro' 'CPPFLAGS=-Wdate-time -D_FORTIFY_SOURCE=2' 'CXXFLAGS=-g -O2 -fdebug-prefix-map=/usr/src/netdata=. -fstack-protector-strong -Wformat -Werror=format-security'
Install type: binpkg-deb
    Binary architecture: x86_64
    Packaging distro:  
Features:
    dbengine:                   YES
    Native HTTPS:               YES
    Netdata Cloud:              YES 
    ACLK:                       YES
    TLS Host Verification:      YES
    Machine Learning:           YES
    Stream Compression:         YES
Libraries:
    protobuf:                YES (system)
    jemalloc:                NO
    JSON-C:                  YES
    libcap:                  NO
    libcrypto:               YES
    libm:                    YES
    tcalloc:                 NO
    zlib:                    YES
Plugins:
    apps:                    YES
    cgroup Network Tracking: YES
    CUPS:                    YES
    EBPF:                    YES
    IPMI:                    YES
    NFACCT:                  YES
    perf:                    YES
    slabinfo:                YES
    Xen:                     NO
    Xen VBD Error Tracking:  NO
Exporters:
    AWS Kinesis:             NO
    GCP PubSub:              NO
    MongoDB:                 NO
    Prometheus Remote Write: YES
Debug/Developer Features:
    Trace Allocations:       NO
```

**Comment by MaxMatti** [comment]
Child log:
```
[...]
2022-11-18 14:10:16: netdata ERROR : PLUGINSD[charts.d] : read failed: end of file (errno 9, Bad file descriptor)
2022-11-18 14:10:16: netdata ERROR : PLUGINSD[charts.d] : '/usr/libexec/netdata/plugins.d/charts.d.plugin' (pid 12959) disconnected after 0 successful data collections (ENDs).
2022-11-18 14:10:16: netdata INFO  : PLUGINSD[charts.d] : '/usr/libexec/netdata/plugins.d/charts.d.plugin' (pid 12959) does not generate useful output but it reports success (exits with 0). Waiting a bit before starting it again..
2022-11-18 14:10:17: netdata INFO  : STREAM_SENDER[devkit-ext-mstaff] : STREAM devkit-ext-mstaff: attempting to connect to '192.168.100.159' (default port: 19999)...
2022-11-18 14:10:17: netdata ERROR : STREAM_SENDER[devkit-ext-mstaff] : Failed to connect to '192.168.100.159', port '19999' (errno 111, Connection refused)
2022-11-18 14:10:17: netdata ERROR : STREAM_SENDER[devkit-ext-mstaff] : STREAM devkit-ext-mstaff [send to 192.168.100.159]: could not connect to parent node at this time.
2022-11-18 14:10:17: cgroup-name.sh: INFO: cgroup 'snap.barrier.barrier' is called 'snap.barrier.barrier'
2022-11-18 14:10:17: /usr/libexec/netdata/plugins.d/cgroup-network ERROR : MAIN : cannot stat() path '/sys/fs/cgroup/cpu,cpuacct/snap.barrier.barrier' (errno 2, No such file or directory)
2022-11-18 14:10:17: /usr/libexec/netdata/plugins.d/cgroup-network ERROR : MAIN : cgroup '/sys/fs/cgroup/cpu,cpuacct/snap.barrier.barrier' does not exist or is not valid.
2022-11-18 14:10:17: netdata ERROR : MAIN : child pid 12999 exited with code 1.
2022-11-18 14:10:22: netdata INFO  : STREAM_SENDER[devkit-ext-mstaff] : STREAM devkit-ext-mstaff: attempting to connect to '192.168.100.159' (default port: 19999)...
2022-11-18 14:10:22: netdata INFO  : STREAM_SENDER[devkit-ext-mstaff] : STREAM devkit-ext-mstaff [send to 192.168.100.159]: initializing communication...
2022-11-18 14:10:22: netdata INFO  : STREAM_SENDER[devkit-ext-mstaff] : STREAM devkit-ext-mstaff [send to 192.168.100.159]: waiting response from remote netdata...
2022-11-18 14:10:22: netdata INFO  : STREAM_SENDER[devkit-ext-mstaff] : STREAM devkit-ext-mstaff [send to 192.168.100.159]: established link with negotiated capabilities: VCAPS HLABELS CLAIM CLABELS FUNCTIONS REPLICATION
2022-11-18 14:10:22: netdata ERROR : STREAM_SENDER[devkit-ext-mstaff] : Clearing stream_collected_metrics flag in charts of host devkit-ext-mstaff
2022-11-18 14:10:22: netdata INFO  : STREAM_SENDER[devkit-ext-mstaff] : STREAM devkit-ext-mstaff [send to 192.168.100.159]: enabling metrics streaming...
2022-11-18 14:10:22: netdata INFO  : PLUGINSD[go.d] : STREAM devkit-ext-mstaff [send]: sending metrics to parent...
2022-11-18 14:10:22: netdata ERROR : STREAM_SENDER[devkit-ext-mstaff] : STREAM devkit-ext-mstaff [send to 192.168.100.159]: connection closed by far end.
2022-11-18 14:10:22: netdata INFO  : STREAM_SENDER[devkit-ext-mstaff] : STREAM devkit-ext-mstaff: attempting to connect to '192.168.100.159' (default port: 19999)...
2022-11-18 14:10:22: netdata ERROR : STREAM_SENDER[devkit-ext-mstaff] : Failed to connect to '192.168.100.159', port '19999' (errno 111, Connection refused)
2022-11-18 14:10:22: netdata ERROR : STREAM_SENDER[devkit-ext-mstaff] : STREAM devkit-ext-mstaff [send to 192.168.100.159]: could not connect to parent node at this time.
2022-11-18 14:10:22: netdata ERROR : MAIN : STREAM devkit-ext-mstaff [send]: not ready - collected metrics are not sent to parent.
2022-11-18 14:10:26: netdata INFO  : PLUGINSD[charts.d] : connected to '/usr/libexec/netdata/plugins.d/charts.d.plugin' running on pid 13066
2022-11-18 14:10:26: charts.d: WARNING: main: command 'curl' is not found in /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin.
2022-11-18 14:10:26: netdata ERROR : PLUGINSD[charts.d] : read failed: end of file (errno 9, Bad file descriptor)
2022-11-18 14:10:26: netdata LOG FLOOD PROTECTION too many logs (201 logs in 20 seconds, threshold is set to 200 logs in 1200 seconds). Preventing more logs from process 'netdata' for 1180 seconds.
2022-11-18 14:10:36: charts.d: WARNING: main: command 'curl' is not found in /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin.
2022-11-18 14:10:46: charts.d: WARNING: main: command 'curl' is not found in /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin.
2022-11-18 14:10:56: charts.d: WARNING: main: command 'curl' is not found in /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin.
2022-11-18 14:11:06: charts.d: WARNING: main: command 'curl' is not found in /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin.
2022-11-18 14:11:16: charts.d: WARNING: main: command 'curl' is not found in /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin.
2022-11-18 14:11:27: charts.d: WARNING: main: command 'curl' is not found in /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin.
2022-11-18 14:11:37: charts.d: WARNING: main: command 'curl' is not found in /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin.
2022-11-18 14:11:47: charts.d: WARNING: main: command 'curl' is not found in /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin.
2022-11-18 14:12:07: alarm-notify.sh: WARNING: Cannot find file '/etc/netdata/health_alarm_notify.conf'.
```

Parent log:
```
[...]
2022-11-18 14:17:01: python.d INFO: plugin[main] : uwsgi[localhost] : check failed
2022-11-18 14:17:01: python.d ERROR: uwsgi[localipv4] : Failed to connect to "127.0.0.1", port 1717, error: [Errno 111] Connection refused
2022-11-18 14:17:01: python.d INFO: plugin[main] : uwsgi[localipv4] : check failed
2022-11-18 14:17:01: python.d ERROR: uwsgi[localipv6] : Failed to connect to "::1", port 1717, error: [Errno 111] Connection refused
2022-11-18 14:17:01: python.d INFO: plugin[main] : uwsgi[localipv6] : check failed
2022-11-18 14:17:01: python.d ERROR: varnish[varnish] : can't locate 'varnishstat' binary or binary is not executable by user netdata
2022-11-18 14:17:01: python.d INFO: plugin[main] : varnish[varnish] : check failed
2022-11-18 14:17:01: python.d ERROR: w1sensor[w1sensor] : [Errno 2] No such file or directory: '/sys/bus/w1/devices/'
2022-11-18 14:17:01: python.d INFO: plugin[main] : w1sensor[w1sensor] : check failed
2022-11-18 14:17:03: netdata FATAL : STREAM_RECEIVER[devkit-ext-mstaff,[192.168.100.186]:55316] : DBENGINE: page_index has a different alignment (page_index refcount is 2, writers is 1). # : Success

/usr/sbin/netdata(+0x87f0d)[0x560062b6ff0d]
/usr/sbin/netdata(+0x285ba5)[0x560062d6dba5]
/usr/sbin/netdata(+0x285d7a)[0x560062d6dd7a]
/usr/sbin/netdata(+0x1b1ac4)[0x560062c99ac4]
/usr/sbin/netdata(+0x73538)[0x560062b5b538]
/usr/sbin/netdata(+0x74847)[0x560062b5c847]
/usr/sbin/netdata(+0x749f3)[0x560062b5c9f3]
/usr/sbin/netdata(+0x1b2654)[0x560062c9a654]
/usr/sbin/netdata(+0x19e596)[0x560062c86596]
/usr/sbin/netdata(+0x2a5029)[0x560062d8d029]
/usr/sbin/netdata(+0x29235c)[0x560062d7a35c]
/usr/sbin/netdata(+0x293388)[0x560062d7b388]
/usr/sbin/netdata(+0x293bae)[0x560062d7bbae]
/usr/sbin/netdata(+0x93887)[0x560062b7b887]
/lib/x86_64-linux-gnu/libpthread.so.0(+0x8609)[0x7f8917c50609]
/lib/x86_64-linux-gnu/libc.so.6(clone+0x43)[0x7f8917827133]
2022-11-18 14:17:03: netdata LOG FLOOD PROTECTION reset for process 'netdata' (prevented 124 logs in the last 4 seconds).
2022-11-18 14:17:03: netdata INFO  : STREAM_RECEIVER[devkit-ext-mstaff,[192.168.100.186]:55316] : EXIT: netdata prepares to exit with code 1...
2022-11-18 14:17:03: netdata INFO  : STREAM_RECEIVER[devkit-ext-mstaff,[192.168.100.186]:55316] : EXIT: cleaning up the database...
2022-11-18 14:17:03: netdata INFO  : STREAM_RECEIVER[devkit-ext-mstaff,[192.168.100.186]:55316] : Cleaning up database [2 hosts(s)]...
2022-11-18 14:17:03: netdata INFO  : STREAM_RECEIVER[devkit-ext-mstaff,[192.168.100.186]:55316] : Cleaning up database of host 'anvajo-nb97.anvajo.local'...
2022-11-18 14:17:03: netdata INFO  : STREAM_RECEIVER[devkit-ext-mstaff,[192.168.100.186]:55316] : Cleaning up database of host 'devkit-ext-mstaff'...
2022-11-18 14:17:03: netdata INFO  : STREAM_RECEIVER[devkit-ext-mstaff,[192.168.100.186]:55316] : Closing context SQLite database
2022-11-18 14:17:03: netdata INFO  : STREAM_RECEIVER[devkit-ext-mstaff,[192.168.100.186]:55316] : Closing SQLite database
2022-11-18 14:17:03: netdata INFO  : STREAM_RECEIVER[devkit-ext-mstaff,[192.168.100.186]:55316] : No statements pending to finalize
2022-11-18 14:17:03: netdata INFO  : STREAM_RECEIVER[devkit-ext-mstaff,[192.168.100.186]:55316] : EXIT: removing netdata PID file '/var/run/netdata/netdata.pid'...
2022-11-18 14:17:03: netdata INFO  : STREAM_RECEIVER[devkit-ext-mstaff,[192.168.100.186]:55316] : EXIT: all done - netdata is now exiting - bye bye...
2022-11-18 14:17:03: apps.plugin ERROR : APPS_READER : Received error on stdin.
EOF found in spawn pipe.
Shutting down spawn server event loop.
Shutting down spawn server loop complete.
2022-11-18 14:17:03: go.d INFO: main[main] received terminated signal (15). Terminating...
2022-11-18 14:17:03: go.d INFO: run[manager] instance is stopped
2022-11-18 14:17:03: go.d INFO: discovery[file manager] instance is stopped
2022-11-18 14:17:03: go.d INFO: build[manager] instance is stopped
2022-11-18 14:17:03: go.d INFO: discovery[manager] instance is stopped
2022-11-18 14:17:03: go.d INFO: state save[manager] instance is stopped
```

**Comment by MrZammler** [comment]
Hi @MaxMatti thanks for the report, we will try to reproduce the issue and find the cause.

**Comment by MaxMatti** [comment]
Trying this with `v1.37.0-91-nightly` does not crash netdata, however I am not able to open the metrics page for the client. Opening `http://localhost:19999/host/devkit-ext-mstaff/` only displays the text `This netdata does not maintain a database for host: devkit-ext-mstaff`. Is there a way to list the available children to ensure that I entered the correct name?

**Comment by ilyam8** [comment]
Check `http://localhost:19999/api/v1/info`

**Comment by MaxMatti** [comment]
Thanks a lot for the help, @ilyam8 ! Unfortunately the child does not show up in the parents `mirrored hosts` section under `http://localhost:19999/api/v1/info`. Should I open a new issue for that?

**Comment by ilyam8** [comment]
@MaxMatti I suggest to re-check your streaming configuration and error.log (`grep -i stream error.log`) before opening an issue.

**Comment by MaxMatti** [comment]
Re-checking did indeed help, thank you so much!
